Week-7/7.5/index7.5.js

```diff
@@ -0,0 +1,19 @@
+function nextGreaterElement(arr) {
+    var s = [];
+    let res = new Array(arr.length);
+
+    for (let i = arr.length - 1; i >= 0; i--) {
+        if (s.length != 0) {
+            while (s.length != 0 && s[s.length - 1] <= arr[i]) {
+                s.pop();
+            }
+        }
+        res[i] = s.length == 0 ? -1 : s[s.length - 1];
+        s.push(arr[i]);
+    }
+    for (let i = 0; i < arr.length; i++)
```

Comment:
just return the array and console log the same. We don't need to iterate and show them one by one